src/dmg2john.c

```diff
@@ -1,4 +1,3 @@
-#if !defined (_MSC_VER) && !defined (__MINGW32__)
```

Comment:
I am not sure if VC will like the code though. I can keep the `_MSC_VER` part intact.